packages/editor-ui/src/__tests__/utils.ts

```diff
@@ -108,3 +111,28 @@ export const createAppModals = () => {
 export const cleanupAppModals = () => {
 	document.body.innerHTML = '';
 };
+
+/**
+ * Typescript helper for mocking pinia store actions return value
+ *
+ * @see https://pinia.vuejs.org/cookbook/testing.html#Mocking-the-returned-value-of-an-action
+ */
+export const mockedStore = <TStoreDef extends () => unknown>(
+	useStore: TStoreDef,
+): TStoreDef extends StoreDefinition<infer Id, infer State, infer Getters, infer Actions>
+	? Store<
+			Id,
+			State,
+			Record<string, never>,
+			{
+				[K in keyof Actions]: Actions[K] extends (...args: infer Args) => infer ReturnT
+					? Mock<Args, ReturnT>
+					: Actions[K];
+			}
+		> & {
+			[K in keyof Getters]: Getters[K] extends ComputedRef<infer T> ? T : never;
+		}
+	: ReturnType<TStoreDef> => {
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	return useStore() as any;
```

Comment:
I think you can omit type casting here to `any`, or if you want to do it, it could be `TStoreDef`

Reply:
I thought just to avoid using eslint disable comment because we're not using any

Reply:
And it is likely doesn't matter since the return type of the function already defined as `ReturnType<TStoreDef>`

Reply:
i agree but it doesn't work :(